Sources/NIOHTTP2/ConnectionStateMachine/ConnectionStreamsState.swift

```diff
@@ -77,10 +77,11 @@ struct ConnectionStreamState {
     /// - parameters:
     ///     - streamID: The ID of the pushed stream.
     ///     - remoteInitialWindowSize: The initial window size of the remote peer.
+    ///     - requestVerb: HTTP Request Method used for different purposes with a web server
```

Comment:
I think we can make this clearer: we can say "the HTTP method used on the request".